lib/ts/recipe/thirdpartyemailpassword/types.ts

```diff
@@ -196,6 +196,8 @@ export type TypeNormalisedInput = {
 export interface RecipeInterface {
     getUserById(input: { userId: string }): Promise<User | undefined>;
 
+    getThirdPartyUsersByEmail(input: { email: string }): Promise<ThirdPartyUser[]>;
```

Comment:
See comment in `lib/ts/recipe/thirdpartyemailpassword/recipeImplementation/index.ts`